source/sdk/math.ooc

```diff
@@ -74,22 +70,8 @@ extend Int {
 		result % divisor
 	}
 	clamp: func (floor: This, ceiling: This) -> This {
-		if (this > ceiling)
-			ceiling
-		else if (this < floor)
-			floor
-		else
-			this
+		this > ceiling ? ceiling : (this < floor ? floor : this)
```

Comment:
No, it's not "too bad", it just takes a bit of getting used to. For a function that does so little work, I guess it's fine.